src/main/java/com/mael/ttt/Turn.java

```diff
@@ -20,31 +23,28 @@ public void placeMark(Player player) {
         board.setCell(player.getMove(board), player.getMark().getString());
     }
 
-    public boolean isNotGameOver() {
-        return !(hasWinner() || checkForFull());
-    }
-
-    private boolean hasWinner() {
-        return checkForWinner(Mark.PLAYER) || checkForWinner(Mark.OPPONENT);
+    public boolean canBePlayed() {
+        return !(hasWinner() || isFull());
```

Comment:
I usually don't like negating methods that return `boolean` values, it requires more context switching than if you'd have methods that express the opposite:

```
private boolean noWinner() {
}
private boolean isNotFull() {
}
public boolean canBePlayed() {
  return noWinner() || isNotFull();
}
```